molecule/driver/docker.py

```diff
@@ -114,7 +117,7 @@ class Docker(base.Base):
         - name: instance
           image: centos:7
           privileged: true
-          volume_mounts:
+          volumes:
```

Comment:
Can we put this unrelated change into another commit?

Reply:
yes please